src/common/collection.c

```diff
@@ -705,6 +705,7 @@ const char *dt_collection_name(dt_collection_properties_t prop)
     case DT_COLLECTION_PROP_LOCAL_COPY:       return _("local copy");
     case DT_COLLECTION_PROP_MODULE:           return _("module");
     case DT_COLLECTION_PROP_ORDER:            return _("module order");
+    case DT_COLLECTION_PROP_RATING:           return _("image rating");
```

Comment:
probably just "rating" here as already done in the code in different places.

Reply:
Agreed: 6877871